staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -52,6 +52,18 @@ func NewNoxuSubresourcesCRD(scope apiextensionsv1beta1.ResourceScope) *apiextens
 				ListKind:   "NoxuItemList",
 			},
 			Scope: scope,
+			Versions: []apiextensionsv1beta1.CustomResourceDefinitionVersion{
```

Comment:
the test passes even without all the other changes in that PR.

Reply:
Working on a fix in the versioning decoder.

If I cannot work it out today, I can do a quickfix for Unstructured. It's not pretty, but will unblock versioning.